src/main/java/com/pwn9/filter/bukkit/BukkitAPI.java

```diff
@@ -61,8 +68,14 @@
     from the main thread while there is an outstanding request.
      */
 
-    BukkitAPI(PwnFilterPlugin p) {
+    @SuppressWarnings({"Unsafe", "WeakerAccess"})
+    protected BukkitAPI(PwnFilterPlugin p) {
         plugin = p;
+        if(p instanceof PwnFilterBukkitPlugin) {
+            audiences = BukkitAudiences.create((PwnFilterBukkitPlugin) p);
```

Comment:
Codacy found an issue: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)